### ISSUE TITLE
Missing `flutter_post_install(installer)` on Flutter 3.3.10

### ISSUE THREAD

**Issue by ahmdmau** [DartBuild/cocoapods-embed-flutter#43]
Hi, hope you are doing well.
When I try to use this plugin using Flutter 3.3.10 it showed an error `flutter_post_install(installer)`, maybe caused by this https://github.com/flutter/flutter/pull/102538. Is it there a way to handle this error?

**Comment by soumyamahunt** [comment]
Yes seems like `flutter_post_install` is no longer needed. Do you want to create a PR fixing this issue?
